google_guest_agent/system_windows.go

```diff
@@ -72,3 +106,50 @@ func deleteRegKey(key, name string) error {
 
 	return k.DeleteValue(name)
 }
+
+func checkWindowsServiceRunning(servicename string) bool {
+	res := runCmdOutput(exec.Command("sc", "query", servicename))
+	return strings.Contains(res.Stdout(), "RUNNING")
+}
+
+func getWindowsServiceImagePath(regKey string) (string, error) {
+	regValue, err := readRegString(regKey, "ImagePath")
+	if err != nil {
+		return "", err
+	}
+	imagePath := strings.Trim(string(regValue), `"`)
+	return imagePath, nil
+}
+
+func parseVersionInfo(psOutput []byte) (versionInfo, error) {
```

Comment:
this doesn't need to be in the windows specific file; do the unit tests against this function work when running the tests on linux?

Reply:
Moved to windows_accounts.go along with the test and removed system_windows_test.go